Cargo.lock

```diff
@@ -6482,10 +6482,11 @@ dependencies = [
 
 [[package]]
 name = "serde"
-version = "1.0.219"
+version = "1.0.224"
```

Comment:
Is it ok that we are choosing a version that is != what is in Cargo.toml?

Reply:
good catch. updated the pr title